src/components/add-movie-form/add-movie-form.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button } from 'components/button';
+import { Input } from 'components/input';
+import React, { FC, FormEvent, useState } from 'react';
+import {
+  AddMovieButtonContainer,
+  AddMovieFormContainer,
+  AddMovieFormInner,
+  AddMovieFormTitle,
+  AddMovieFormWrapper,
+  CloseButton,
+} from './styles';
```

Comment:
do not like the 'styles' name because it actually are components. Confused me a bit. I would change name.